app/app.py

```diff
@@ -16,6 +18,23 @@
 config = Configuration(config_path)
 
 
+def transform_time(time_string):
+    # now, today, eod, today-week, today-month
+
+    current = datetime.datetime.now()
+    if time_string == 'now':
+        result = current
+    if time_string == 'today':
+        result = current.replace(current.year, current.month, current.day, 0, 0, 0, 0)
+    if time_string == 'eod':
+        result = current.replace(current.year, current.month, current.day, 23, 59, 59)
+    if time_string == 'today-week':
+        result = current - datetime.timedelta(days=7)
```

Comment:
I would prefer to just not support these at all rather than special casing some of them. It might lead to unexpected behaviour if sometimes these work and sometimes they don't. We can try to add proper support for these via a separate input text box later. Please remove this function for now.

Reply:
I think you might have missed this comment :)

Reply:
Sorry I forgot to explain :-( I got your ideas and I thought it made sense, so I removed the custom range from user interface in `recipe.js`. However, this function should be kept because it transforms the default value (eod, now,...) which is hard-code in `COMMON_CONTEXT`. I think I should implement a textbox to avoid confusion in this situation.

Reply:
Oh I see, thanks for the explanation! Yes, I agree this makes sense for now.